README.md

```diff
@@ -27,27 +27,21 @@ Checking how many of these guesses are correct, the success rates of the differe
 
 ## Setup and installation
 
+Anonymeter requires Python 3.8.x, 3.9.x or 3.10.x (this limitation is due to Numba usage).
```

Comment:
Is the limitation about not being able to run with Python 3.11? I'd be more explicit about that (e.g., “Python 3.11 is not supported (yet) due to Numba usage”).

Reply:
It's also the limitation of not being able to run with Python 3.7 :) Numba supports only 3.8, 3.9 and 3.10 in the current release

Reply:
I can just remove the numba mention to avoid confusion